include/conf/load.hpp

```diff
@@ -37,17 +38,50 @@ namespace color_coded
       }
     }
 
-    inline args_t load(std::string const &file, std::string const &filetype)
+    inline args_t load_compilation_database(std::string const &file, std::string filename)
     {
-      if(file.empty())
-      { return defaults(filetype); }
+      std::string str;
+      auto database_ptr = ::clang::tooling::JSONCompilationDatabase::loadFromFile(file, str);
```

Comment:
This `database_ptr` should be const and use direct initialization. Does this need to be cleaned up somewhere?